lib/better_assert_difference/version.rb

```diff
@@ -1,3 +1,3 @@
 module BetterAssertDifference
-  VERSION = '0.1.13'
+  VERSION = '0.2.0'
```

Comment:
I would have bumped the major due to the breaking change, but as we're not in 1.0 yet, I'm not sure what are SemVer guidelines here, so I chose to just increase the minor.